SPECS/PyYAML/PyYAML.spec

```diff
@@ -51,10 +52,14 @@ chmod a-x examples/yaml-highlight/yaml_hl.py
 %files
 %defattr(-,root,root,-)
 %license LICENSE
-%doc PKG-INFO README examples
+%doc CHANGES README.md examples
 %{python3_sitelib}/*
 
 %changelog
+* Thu Feb 01 2024 Henry Li <lihl@microsoft.com> - 6.0.1-1
+- Upgrade to version 6.0.1
+- Fix Source0
+
```

Comment:
can you also mention that the debuginfo package is now disabled

Reply:
Fixed